test/test.c

```diff
@@ -1645,8 +1647,15 @@ static void ShowHelp()
 "       --crop   <top:bottom:left:right>\n"
 "                           Set picture cropping in pixels\n"
 "                           (default: automatically remove black bars)\n"
-"       --loose-crop        Always crop to a multiple of the modulus\n"
+"       --loose-crop        Automatic Cropping but using conservative algorithm. \n"
```

Comment:
Keeping `--loose-crop` and `--no-loose-crop` seems wrong since we're removing the `PresetLooseCrop` key. (FWIW, I would also support changing all instances of loose to conservative under the hood. *runs*)

From a user perspective, it would be good to handle `--crop=auto|conservative|none|x:x:x:x`, or if that's a PITA, perhaps leave `--crop` alone and add `--crop-mode=auto|conservative|none` in place of the loose arguments. Either would also provide a bit nicer way to disable cropping as we're doing in the UIs (adding `None` as an option).